getting_started/data_backup.md

```diff
@@ -0,0 +1,220 @@
+
+#### Google Drive
+
+After completing [steps 1 - 3 above](#configuring-rclone-remotes): you will be prompted to enter a [`client_id`](https://rclone.org/drive/#making-your-own-client-id)
+
+1. When prompted from a [`client_id`](https://rclone.org/drive/#making-your-own-client-id), flow the instruction on the linked page.
+    - *You must use a non-CMU account* due to institutional restrictions.
+    - You *can* use this client id to sync with your Andrew GDrive account.
+    - You can skip this step, but performance may be limited.
+
+2. Paste your `client_id` and then `secret` when prompted by `rclone configure`.
+3. The default options are fine until reaching:
+4. For "Use auto config?" enter `n` if configuring rclone on arjuna (Or another headless machine). Otherwise, enter `y`
+5. Open the link in a browser (If you entered `y` above a browser window should open).
+    - Choose the Google account where you want to back up (Ie. your Andrew Account).
+    - If using a personal client id, you may get a warning that "Google hasn't verified this app", click "Continue".
+
+6. After authenticating, copy the code and paste it into the rclone prompt.
+7. The remaining defaults should be fine.
+
+#### Box
+
+1. The default options are fine until reaching:
+2. For "Use auto config?" enter `n` if configuring rclone on arjuna (Or another headless machine). Otherwise, enter `y`.
+3. Follow the instructions to authenticate with Box.
+4. The remaining defaults should be fine.
+
+### Syncing Files with `rclone`
+
+To [sync](https://rclone.org/commands/rclone_sync/) a folder to a remote destination, use the `rclone sync` command:
+
+```bash
+# Local (Or from a teminal on Arjuna) to Remote
+rclone sync -P path/to/folder remote:path/to/folder
+
+# Remote to Local (Or from a teminal on Arjuna)
+rclone sync -P remote:path/to/folder path/to/folder
+```
+
+The `-P` flag is used to enable progress reporting and will display real-time progress statistics. You may get a bunch of warnings about symlinks not being copied. To copy symlinked files and folders, use the `--copy-links` flag.
+
+## Automated Backups
+
+A generic cron job to run backups is provided below, to use it do the following:
+
+1. Configure [rclone](#transfers-to-cloud-storage-rclone) for your cloud storage provider on Arjuna.
+2. Create a file `~/backup.sh` with the [indicated contents](#contents-of-backupsh). Remember to replace `remote` with the name of your remote
+3. Run `crontab -e`
+4. In the editor that opens, add the following line to the bottom of the file:
+   `0 0 1 * * bash ~/backup.sh`
+5. Trigger the first backup manually: `bash ~/backup.sh`
+6. The next back will occur at 0:00 on the first day of the month.
+   *CHECK THAT THIS IS HAPPENS!*
+
+> Users are responsible for backing up their data, verifying their backups,
+> and the operation of any backup scripts. *INCLUDING THIS ONE*!
```

Comment:
"operation of any backup scripts" I think was meant to mean "verifying the operation of any backup scripts or something"